### DISCUSSION TITLE
Block temporary mailbox

### DISCUSSION THREAD

**Discussion by haig233** [postalserver/postal#3040] [Help with using Postal]
When I use a postal server, my users often send emails to some temporary mailboxes, such as https://temp-mail.org/. I want to block these temporary mailboxes. How can I do this? There is no relevant setting in the configuration file.

**Comment by willpower232** [comment]
Postal does not care about the addresses people use unless they are proven hard fails. You should implement this block in whatever sends the messages to Postal. If that is your users, then you would have to educate them.

Technically you may be able to block access to that domain from the server running Postal but that depends on how your server is set up.
